idseq_dag/steps/generate_taxid_fasta.py

```diff
@@ -55,6 +55,9 @@ def run(self):
         input_fa.close()
         output_fa.close()
 
+    def count_reads(self):
```

Comment:
Should change to the single method referred in previous comment.

Reply:
i think this should be implemented based on the context.